### DISCUSSION TITLE
autodiff not found by cmake

### DISCUSSION THREAD

**Discussion by khavernathy** [autodiff/autodiff#216] [Q&A] [answered]
Hi, I'm trying to use `autodiff` as a dependency in an ongoing project. As instructed I added 

```
find_package(autodiff)
target_link_libraries(mypackagename autodiff::autodiff)
```
to my `CMakeLists.txt`. When I try to build, I get

```
CMake Warning at CMakeLists.txt:165 (find_package):
  By not providing "Findautodiff.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "autodiff",
  but CMake did not find one.

  Could not find a package configuration file provided by "autodiff" with any
  of the following names:

    autodiffConfig.cmake
    autodiff-config.cmake

  Add the installation prefix of "autodiff" to CMAKE_PREFIX_PATH or set
  "autodiff_DIR" to a directory containing one of the above files.  If
  "autodiff" provides a separate development package or SDK, be sure it has
  been installed.
  ```
  
  Am I doing something wrong? Please assist!

**Comment by allanleal** [comment, marked as answer]
Hi, you need to ensure two things:

- you have autodiff installed somewhere in your system (say, `/home/user/libs/autodiff`)
- `CMAKE_PREFIX_PATH` knows that path (if not in a default path), something like: `cmake .. -DCMAKE_PREFIX_PATH=/home/user/libs/`.

Please research as this is a cmake related issue, not `autodiff`.
